sites/automationworld/redirect-handler.js

```diff
@@ -0,0 +1,15 @@
+const redirect = (from, pattern) => pattern.exec(from);
```

Comment:
Rename this function to be declarative of what it's doing/returning... it's not actually redirecting anything. Use something like `match` or `exec`